Hit "cp: illegal option -- t" while making etcd image in macOS

**Is this a BUG REPORT or FEATURE REQUEST?**:

 /kind bug

**What happened**:
hit "cp: illegal option -- t" while making etcd image in macOS

**What you expected to happen**:
etcd image created correctly

**How to reproduce it (as minimally and precisely as possible)**:
```
$ make
# Copy the content in this dir to the temp dir,
# without copying the subdirectories.
find ./ -maxdepth 1 -type f | xargs cp -t /var/folders/mt/xcthpky9237d5drdp2hv76fr0000gn/T/tmp.MnhT8IPQ
cp: illegal option -- t
usage: cp [-R [-H | -L | -P]] [-fi | -n] [-apvXc] source_file target_file
       cp [-R [-H | -L | -P]] [-fi | -n] [-apvXc] source_file ... target_directory
make: *** [build] Error 123
```

**Anything else we need to know?**:

**Environment**: macOS
- Kubernetes version (use `kubectl version`): master branch code
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`): Darwin Kernel Version 17.2.0: Fri Sep 29 18:27:05 PDT 2017; root:xnu-4570.20.62~3/RELEASE_X86_64 x86_64
- Install tools:
- Others:

--- a/cluster/images/etcd/Makefile
+++ b/cluster/images/etcd/Makefile
@@ -53,7 +53,7 @@ endif
 build:
 	# Copy the content in this dir to the temp dir,
 	# without copying the subdirectories.
-	find ./ -maxdepth 1 -type f | xargs cp -t $(TEMP_DIR)
+	find ./ -maxdepth 1 -type f | xargs -I {} cp {} $(TEMP_DIR)
 
 	# Compile attachlease
 	docker run -i -v $(shell pwd)/../../../:/go/src/k8s.io/kubernetes -v $(TEMP_DIR):/build -e GOARCH=$(ARCH) golang:$(GOLANG_VERSION) \

